src/Razor/tools/RazorSyntaxGenerator/AbstractFileWriter.cs

```diff
@@ -347,6 +347,7 @@ protected static bool IsKeyword(string name)
                 case "event":
                 case "enum":
                 case "unsafe":
+                case "text":
```

Comment:
Why this change? `RazorSyntaxGenerator` is used to automatically generate files inside https://github.com/dotnet/aspnetcore-tooling/tree/master/src/Razor/src/Microsoft.AspNetCore.Razor.Language/Syntax/Generated which are pure C#. `text` isn't a keyword there.

Reply:
Removed, thanks.